lib/rubocop/cop/rspec/mixin/top_level_describe.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Helper methods for top level describe cops
+      module TopLevelDescribe
```

Comment:
I believe we've got rid of this one.

Reply:
You’re absolutely right. I remember thinking about it during the rebase, but I must have forgot. Let’s hope it’s the only thing I forgot 😱